### ISSUE TITLE
Adjust the mobile menu button for accessibility: 3

### ISSUE THREAD

**Issue by paregorios** [isawnyu/isaw.web#266]
_Epic: #265_

> The mobile menu icon should be given the aria attribute role = button. It should also be given an attribute aria-pressed = false, which will be programmatically toggled to match the state of the links being visible. This will need to work as a toggle element with only a keyboard as a navigational tool. The icon should open the menu on enter or spacebar keypresses.

WCAG 2.0 Compliance: [4.1.2 Name, Role, Value](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html)

> For all user interface components (including but not limited to: form elements, links and components generated by scripts), the name and role can be programmatically determined; stated, properties, and values that can be set by the user can be programmatically set; and notification of changes to these items is available to user agents, including assistive technologies. (Level A)

![screen shot 2018-04-27 at 2 55 06 pm](https://user-images.githubusercontent.com/263285/39382424-0753f73a-4a2b-11e8-98d4-019fe7dee9fd.png)

Traceability: [Microassist Report 1 > WCAG Compliance > Name, Role, Value](https://docs.google.com/document/d/10cfUFx-o_kSGBFKCe4MtL45cyc-c-Gwaz3Dxlrp8Qgg/edit#bookmark=id.t8cmbkue4bcm)

**Comment by paregorios** [comment]
Please merge to master and deploy to production.
